playbooks/test_deploy_cukinia_tests.yaml

```diff
@@ -3,12 +3,16 @@
 
 ---
 - name: Deploy Cukinia's tests
-  hosts: cluster_machines
+  hosts: hypervisors
   tasks:
     - name: Copy Cukinia's tests
       copy:
         src: ../src/cukinia-tests/cukinia
         dest: /etc/
+    - name: Copy cukinia.conf
+      template:
```

Comment:
cukinia-hypervisor.conf.j2 is not a template but regular file. Use copy instead.